spec/02-integration/05-proxy/11-error_default_type_spec.lua

```diff
@@ -42,6 +42,15 @@ describe("error_default_type", function()
 
   describe("request `Accept` is missing", function()
     describe("(default)", function()
+      setup(function()
+        assert(helpers.kong_exec("reload", {
+          prefix             = helpers.test_conf.prefix,
+          nginx_conf         = "spec/fixtures/custom_nginx.template",
+        }))
+
+        ngx.sleep(RELOAD_DELAY)
```

Comment:
It does seem to me like we need to reload Kong here, since it is already being started with the default value for `error_default_type`? Especially considering the how flaky this test suite is by using `reload` (See #2805).